src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -106,31 +106,37 @@ export const TxDetails = ({ transaction, actions }: TxDetailsProps): ReactElemen
 
   return (
     <TxDetailsContainer>
-      <div className={cn('tx-summary', { 'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED' })}>
-        <TxSummary txDetails={data} />
-      </div>
-      <div
-        className={cn('tx-details', {
-          'no-padding': isMultiSendTxInfo(data.txInfo),
-          'not-executed': !data.executedAt,
-          'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED',
-        })}
-      >
-        <TxDataGroup txDetails={data} />
-      </div>
-      <div
-        className={cn('tx-owners', {
-          'no-owner': txLocation !== 'history' && !actions?.isUserAnOwner,
-          'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED',
-        })}
-      >
-        <TxOwners txDetails={data} />
+      <div>
+        <div
+          className={cn('tx-details', {
+            'no-padding': isMultiSendTxInfo(data.txInfo),
+            'not-executed': !data.executedAt,
+            'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED',
```

Comment:
A nice way to clean this us a bit is to move some of these conditions into variables as they are repeated, e.g.

```ts
const willBeReplaced = transaction.txStatus === 'WILL_BE_REPLACED'
const isHistorical = txLocation === 'history'
```